api/v1/storagecluster_types.go

```diff
@@ -137,11 +140,41 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageKind specifies a kind of the external storage
+type ExternalStorageKind string
+
+const (
+	// KindOCS specifies a "ocs" kind of the external storage
+	KindOCS ExternalStorageKind = "ocs"
+
+	// KindRHCS specifies a "rhcs" kind of the external storage
+	KindRHCS ExternalStorageKind = "rhcs"
+)
+
 // ExternalStorageClusterSpec defines the spec of the external Storage Cluster
 // to be connected to the local cluster
 type ExternalStorageClusterSpec struct {
-	// +optional
+	//+kubebuilder:default:=true
+	// Enable enables the external mode
 	Enable bool `json:"enable,omitempty"`
+
+	//+kubebuilder:default:=rhcs
+	// StorageProviderKind describes the kind of the external storage
+	StorageProviderKind ExternalStorageKind `json:"storageProviderKind,omitempty"`
```

Comment:
Since only 2 values are acceptable, enum(ocs,rhcs) might be a good validation to add here.

Reply:
done